pkg/cri/server/sandbox_run.go

```diff
@@ -326,6 +334,9 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		}
 
 		sandboxCreateNetworkTimer.UpdateSince(netStart)
+		span.AddEvent("finished pod network setup",
+			tracing.SpanEventAttribute(tracing.Attribute("pod.network.setup.duration", time.Since(netStart).String())),
```

Comment:
How does it access the current span?

Reply:
I am getting the span from the context in L64?